models/classes/runner/time/TimerAdjustmentService.php

```diff
@@ -43,11 +43,12 @@ class TimerAdjustmentService extends ConfigurableService implements TimerAdjustm
     public function increase(
         TestSession $testSession,
         int $seconds,
+        string $type = self::TYPE_TIME_ADJUSTMENT,
```

Comment:
`self::TYPE_TIME_ADJUSTMENT` is a default type, since it is used as a default argument value, correct?

Reply:
Yes. Before that we didn't have this parameter and it was possible to call this method with just two parameters. Therefore I added it as default value do don't break these method calls.